src/EnergyPlus/WaterCoils.cc

```diff
@@ -1050,7 +1050,7 @@ void InitWaterCoil(EnergyPlusData &state, int const CoilNum, bool const FirstHVA
     Real64 EnthCorrFrac(0.0); // enthalpy correction factor
     Real64 TempCorrFrac(0.0); // temperature correction factor
 
-    auto &Node = state.dataLoopNodes->Node;
+    /*auto &Node = state.dataLoopNodes->Node;*/
```

Comment:
I don't think we are supposed to use C-style comments, they interfere with some of the code analysis scripts.

Reply:
Removed.